### PR TITLE
Design Refactor - Cleaning up Code

### PR DESCRIPTION
**Pull request by k-erby**
Probs a WIP.

Here's what's changed:
- reorganized `animations.scss`
- cleaned up some unused rules and stuff
- changed button colors
- orbit on logo now occurs AFTER all parts of loaded
  - also, new animation ASTEROID makes orbit fill go from light blue --> white
- added padding to the FAQ question headers
- changed`main.scss` to import all the `scss` files. `gulpfile.js` now compiles JUST the `main.scss` file to be a `css` file, for very basic fast loading.

Link to vid of walkthrough refactor:
https://drive.google.com/open?id=1PN_IklLarM28n5LfDXeRG4GFDCr-moP6

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -74,14 +74,21 @@
           <div class="row">
             <div class="col-12">
 
-              <!-- Nasa-HackUVic Logo -->
-              <figure class="img">
+              <!-- Nasa-HackUVic Logo
+                The order of the g path MUST be written as it is below. This is due to SVGs not being NICE and allowing
+                the use of the z-index property. SVGs calculate their z-index based on how the DOM renders the parts of
+                the SVGs. This is why the Swooshes and the HackUVic words and everything are split into multiple parts
+                and seem very disconnected when you're reading them straight on. I sold my soul for this knowledge.
```
Comment (FlyteWizard) [inline index.html:81]: Thnank you for your soul contribution XD

File: scss/main.scss [hunk 2]
```diff
@@ -4,6 +4,7 @@
 // -----------------------------------------------------------
 
 @import "./branding.scss";
+@import "./animations.scss";
```
Comment (FlyteWizard) [inline scss/main.scss:7]: 👍

**Review by FlyteWizard — APPROVED** [review]
Everything looks good, and the preview looks good 🍡 

Approved 🌮

File: index.html (hunk 1 above)
Comment (k-erby) [reply index.html:81]: <3 <3 <3 <3 <3
It was worth it? Thank you for your appreciation. :'(

**Review by amyhanv — APPROVED** [review]
Amazing

**Pull request merged**